x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/host_isolation_exceptions.ts

```diff
@@ -212,21 +212,15 @@ export default function ({ getService }: FtrProviderContext) {
           it(`[${apiCall.method}] if entry is valid`, async () => {
             const body = apiCall.getBody();
 
-            await supertest[apiCall.method](apiCall.path)
-              .set('kbn-xsrf', 'true')
-              .send(body)
-              .expect(200);
-          });
-
-          it(`[${apiCall.method}] and accept all OSs for os_types`, async () => {
-            const body = apiCall.getBody();
-
             body.os_types = ['linux', 'windows', 'macos'];
```

Comment:
I still think this is not needed here - the `os_types` is already an array of 3 OSs, right? so non need to set it.